### PR TITLE
Update English translations

### PR DESCRIPTION
**Pull request by angus-g**
Just some phrasing changes to make the English translations sound more natural.

### CODE REVIEW

File: conf/locale/locale_en-US.ini [hunk 1]
```diff
@@ -930,7 +930,7 @@ config.db_ssl_mode_helper = (for "postgres" only)
 config.db_path = Path
 config.db_path_helper = (for "sqlite3" and "tidb")
 config.service_config = Service Configuration
-config.register_email_confirm = Require E-mail Confirmation
+config.register_email_confirm = Require email confirmation
```
Comment (Unknwon) [inline conf/locale/locale_en-US.ini]: This one is on purpose, please revert this line.

Comment (angus-g) [inline conf/locale/locale_en-US.ini]: Whoops, good catch. How about `Require Email Confirmation` to keep the correct casing but still make the e-mail -> email change?

Comment (Unknwon) [inline conf/locale/locale_en-US.ini]: Yes, please :wink:

**Comment by Unknwon** [comment]
LGTM.

**Pull request merged**